packages/blade/.storybook/react/preview.js

```diff
@@ -43,6 +43,7 @@ export const parameters = {
 
 export const decorators = [
   (Story, context) => {
+    hiddenStoryStyle.disabled = context.globals.showInternalComponents
```

Comment:
Instead of mutating this imported module here my recommendation would be to instead expose a function from `manager` that toggles the flag for you. Keeps the logic contained within the `manager` module.

Reply:
@divyanshu013 makes sense. Updated PR accordingly.